index.html

```diff
@@ -11,7 +11,10 @@
 
 {% include projects.html %}
 
+{% include map.html %}
+
 {% include footer.html %}
+
 </body>
 
 </html>
```

Comment:
This two trailing </body> </html> tags aren't needed, as they are already in the layout used by index.html